cross-chain/arbitrum/test/utils/upgrades.ts

```diff
@@ -0,0 +1,25 @@
+import { Deployment } from "hardhat-deploy/types"
+import { helpers } from "hardhat"
+
+import type { FactoryOptions } from "hardhat/types"
+import type { Contract } from "ethers"
+import type { UpgradeProxyOptions } from "@openzeppelin/hardhat-upgrades/src/utils/options"
+
+export interface UpgradesUpgradeOptions {
+  contractName?: string
+  initializerArgs?: unknown[]
+  factoryOpts?: FactoryOptions
+  proxyOpts?: UpgradeProxyOptions
+}
```

Comment:
I am confused why we need to redeclare this type. Can't we remove `upgrades.ts` and just call `helpers.upgrades.upgradeProxy` directly from unit tests?

Reply:
You're right, we can simplify it even more [c95995d](https://github.com/keep-network/tbtc-v2/pull/554/commits/c95995d8640b317f2b72658831f2ea041e45380e)